policy_test.go

```diff
@@ -25,19 +25,19 @@ func TestCalcMinRequiredTxRelayFee(t *testing.T) {
 			"zero value with default minimum relay fee",
 			0,
 			defaultMinRelayTxFee,
-			int64(defaultMinRelayTxFee),
+			0,
```

Comment:
This should technically be `int64(defaultMinRelayTxFee)` as it was, but really I see no point in having a test for it since a 0-byte transaction is not possible.

I better test, in my opinion, would be:

```Go
		{
			// Ensure combination of size and fee that are less than 1000
			// bytes produce a non-zero fee.
			"250 bytes with relay fee of 3",
			250,
			3,
			3,
		},
```